app/templates/events/view/tickets/orders.hbs

```diff
@@ -113,7 +113,7 @@
               </div>
             </td>
             <td>
-              US $ {{order.amount}}
+              {{t 'US'}}${{order.amount}}
```

Comment:
Don't put translation helpers for all strings blindly... We'll not be translating currency codes.